upgrading/blue_green_deployments.adoc

```diff
@@ -260,7 +260,7 @@ new image streams.
 Major releases of {product-title} (and sometimes asynchronous errata updates)
 introduce new image streams for builder images for users of Source-to-Image
 (S2I). Upon import, any builds or deployments configured with
-xref:../../dev_guide/builds/triggering_builds.adoc#image-change-triggers[image
+xref:../dev_guide/builds/triggering_builds.adoc#image-change-triggers[image
```

Comment:
what does Warming mean?

Reply:
Added a line later in the section: The green
nodes are then considered _warmed_, and everything else can be moved over using
node evacuation in a later step, proceeding more quickly as a result.